processes/releasing.md

```diff
@@ -3,64 +3,27 @@
 This document details the high-level process of creating a Metal3 release, and
 the post-release actions required to add new release branches to the CI.
 
-Note the [TODO](#releasing-process-todo) section in the end.
-
 ## Process
 
 High-level release process for a major/minor release. For patch releases, only
 release the relevant versions.
 
-**NOTE**: Ironic-image (via upstream Ironic) has different release cadence as
-Metal3 repos. It is not always possible to release them synced.
+**NOTE**: [Ironic-image](https://github.com/metal3-io/ironic-image) (via
+upstream Ironic) and
+[IrSO](https://github.com/metal3-io/ironic-standalone-operator) (linked to
+Ironic-image releases) have different release cadence as CAPM3, BMO and IPAM
+repos.
 
-1. [Release ironic-image](https://github.com/metal3-io/ironic-image/blob/main/docs/releasing.md)
+1. [Release Ironic-image](https://github.com/metal3-io/ironic-image/blob/main/docs/releasing.md)
+1. [Release IrSO](https://github.com/metal3-io/ironic-standalone-operator/blob/main/docs/releasing.md)
 1. [Release IPAM](https://github.com/metal3-io/ip-address-manager/blob/main/docs/releasing.md)
 1. [Release BMO](https://github.com/metal3-io/baremetal-operator/blob/main/docs/releasing.md)
 1. [Release CAPM3](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/releasing.md)
-1. [Tag Mariadb image](#tag-mariadb-image)
 1. [Update Metal3 dev-env and CI to support new releases](#update-metal3-dev-env-and-ci-to-support-new-releases)
 
 **NOTE**: Always follow release documentation from the respective main branch.
 Release documentation in release branches may be outdated.
 
-## Tag Mariadb image
```

Comment:
I think this is still technically needed for legacy patch branches and for the upcoming 1.11 CAPM3. So this can be removed after 1.11 will be EOL.

Reply:
We have had once single actual change in mariadb since 2023, but technically you're correct. I added NOTE that it will  not apply to beyond v1.11.